alibi_detect/cd/lsdd.py

```diff
@@ -134,12 +140,17 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray]:
         """
         return self._detector.score(x)
 
-    def get_config(self) -> dict:
+    def get_config(self) -> dict:  # Needed due to need to unnormalize x_ref
         """
         Get the detector's configuration dictionary.
 
         Returns
         -------
         The detector's configuration dictionary.
         """
-        return self._detector.get_config()
+        cfg = super().get_config()
+        # Unnormalize x_ref
+        if self._detector.preprocess_at_init or self._detector.preprocess_fn is None \
+                or self._detector.x_ref_preprocessed:
+            cfg['x_ref'] = self._detector._unnormalize(cfg['x_ref'])
```

Comment:
Rather confused by this, what happens if there is a user pre-processing function that is different from `_normalize`? Won't we end up with `x_ref` that is completely different from the original `x_ref`?